backend/config/config.py

```diff
@@ -4,6 +4,9 @@
 import multiprocessing
 import multiprocessing.dummy
 
+ONE_MILE_IN_METERS = 1609.344
```

Comment:
You could also import this from distances to avoid duplication and issue sif changes?
Althgough it hopefully shouldnt change too much ;)

Reply:
Yeah, agreed. I'm wary of having a circular import loop though.